app/templates/components/map-from-id.hbs

```diff
@@ -1,3 +1,16 @@
-{{id}}
+{{#jane-maps
+  id='map'
+  sources=sources
+  initOptions=(hash style='mapbox://styles/mapbox/light-v9'
+                    zoom=zoom
+                    center=center
+                    hash=true)
+  mapLoaded=(action 'handleMapLoad')
+  as |map|}}
+{{/jane-maps}}
 
-{{yield}}
\ No newline at end of file
+{{#if noNarrative}}
```

Comment:
IMO I think reversing the logic of this would make it easier to read `{{unless isNarrative 'once upon a time'}}` - nbd, just food for thought